app/templates/measure-concepts/index.hbs

```diff
@@ -32,7 +32,7 @@
       <td>{{row.description}}</td>
       <td>
         <LinkTo @route="measure-concepts.measure-concept" @model={{row}}>
```

Comment:
I would add the `.au-c-link` class to the link (haven't checked how it looks now but that's the class they use in Appuniversum).